src/core_plugins/elasticsearch/lib/cluster.js

```diff
@@ -85,8 +85,12 @@ export default class Cluster {
     };
   }
 
+  getConfig(path) {
```

Comment:
What about just making it a getter named config?

Are we using the path anywhere?